composer.json

```diff
@@ -19,21 +19,22 @@
     ],
     "require": {
         "php": ">=7.4",
-        "symfony/framework-bundle": ">=5.2"
+        "symfony/framework-bundle": "^5.2"
```

Comment:
Is that on purpose? Previous constraint allowed 6.x also

Reply:
sry, again was not on purpose